src/hearings/components/hearing-confirmation/hearing-confirmation.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-panel govuk-panel--confirmation">
+      <h1 class="govuk-panel__title" [innerHtml]="heading"></h1>
+      <div class="govuk-panel__body" [innerHtml]="headingDescription"></div>
+    </div>
+    <div>
+      <h2 class="govuk-heading-m" [innerHtml]="subheading"></h2>
+      <div class="govuk-body" [innerHtml]="subheadingDescription"></div>
+      <div class="govuk-body" [innerHtml]="additionalDescription"></div>
```

Comment:
It doesn't need to use [innerHtml] here, we can directly pass with {{}} inside the html element.

Reply:
`additionalDescription` has to be innerHtml as the input contains `<br>`